File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/PHP/02.More Exercises/GamingStore.php
<?php
$balance = doubleval(readline());

$totalPrice = 0;
while (true){
    $input = readline();

    if ($input === "Game Time") {
        break;
    }
    $gamePrice = 0;

    if ($input === "OutFall 4") {
        $gamePrice = 39.99;
    } else if ($input === "CS: OG") {
        $gamePrice = 15.99;
    } else if ($input === "Zplinter Zell") {
        $gamePrice = 19.99;
    } else if ($input === "Honored 2") {
        $gamePrice = 59.99;
    } else if ($input === "RoverWatch") {
        $gamePrice = 29.99;
    } else if ($input === "RoverWatch Origins Edition") {
        $gamePrice = 39.99;
    } else {
        echo "Not Found".PHP_EOL;
        continue;
    }

    if ($gamePrice > $balance) {
        echo "Too Expensive".PHP_EOL;
        continue;
    }

    $balance -= $gamePrice;
    $totalPrice += $gamePrice;

    echo "Bought $input".PHP_EOL;

    if ($balance === 0) {
        echo "Out of money!".PHP_EOL;
    }
}

$totalPrice = number_format($totalPrice, 2, '.', '');

echo "Total spent: $$totalPrice. Remaining: $$balance";

?>

File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/Java/More Exercises/src/GamingStore.java
import java.util.Scanner;
public class GamingStore {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double balance = Double.parseDouble(scanner.nextLine());
        double totalPrice = 0;

        while (true) {
            String input = scanner.nextLine();
            if (input.equals("Game Time")) {
                break;
            }
            double gamePrice = 0;

            switch (input) {
                case "OutFall 4":
                    gamePrice = 39.99;
                    break;
                case "CS: OG":
                    gamePrice = 15.99;
                    break;
                case "Zplinter Zell":
                    gamePrice = 19.99;
                    break;
                case "Honored 2":
                    gamePrice = 59.99;
                    break;
                case "RoverWatch":
                    gamePrice = 29.99;
                    break;
                case "RoverWatch Origins Edition":
                    gamePrice = 39.99;
                    break;
                default:
                    System.out.println("Not Found");
                    continue;
            }

            if (gamePrice > balance) {
                System.out.println("Too Expensive");
                continue;
            }

            balance -= gamePrice;
            totalPrice += gamePrice;

            System.out.printf("Bought %s\n", input);

            if (balance == 0) {
                System.out.println("Out of money!");
            }
        }

        System.out.printf("Total spent: $%.2f. Remaining: $%.2f", totalPrice, balance);
    }
}
